test/aqua.jl

```diff
@@ -1,27 +1,35 @@
 using Test
-import ClimaCore
+using ClimaCore
 using Aqua
 
-@testset "Aqua tests - unbound args" begin
+@testset "Aqua tests (performance)" begin
     # This tests that we don't accidentally run into
     # https://github.com/JuliaLang/julia/issues/29393
-    Aqua.test_unbound_args(ClimaCore)
+    # Aqua.test_unbound_args(ClimaCore)
+    ua = Aqua.detect_unbound_args_recursively(ClimaCore)
+    @test length(ua) == 0
 
     # See: https://github.com/SciML/OrdinaryDiffEq.jl/issues/1750
     # Test that we're not introducing method ambiguities across deps
     ambs = Aqua.detect_ambiguities(ClimaCore; recursive = true)
     pkg_match(pkgname, pkdir::Nothing) = false
     pkg_match(pkgname, pkdir::AbstractString) = occursin(pkgname, pkdir)
     filter!(x -> pkg_match("ClimaCore", pkgdir(last(x).module)), ambs)
-    for method_ambiguity in ambs
-        @show method_ambiguity
-    end
-    # If the number of ambiguities is less than the limit below,
-    # then please lower the limit based on the new number of ambiguities.
-    # We're trying to drive this number down to zero to reduce latency.
-    @info "Number of method ambiguities: $(length(ambs))"
+
+    # Uncomment for debugging:
+    # for method_ambiguity in ambs
+    #     @show method_ambiguity
+    # end
     @test length(ambs) ≤ 15
+end
 
-    # returns a vector of all unbound args
-    # ua = Aqua.detect_unbound_args_recursively(ClimaCore)
+@testset "Aqua tests (additional)" begin
+    Aqua.test_undefined_exports(ClimaCore)
+    Aqua.test_stale_deps(ClimaCore)
+    Aqua.test_deps_compat(ClimaCore)
+    Aqua.test_project_extras(ClimaCore)
+    # Aqua.test_project_toml_formatting(ClimaCore) # failing
+    Aqua.test_piracy(ClimaCore) # failing
```

Comment:
Is this still failing? It looks like all the tests pass on both buildkite and GHA

Reply:
Ah, right, I forgot to remove-- I think I recently fixed it, actually